.github/workflows/common-flow-host.yml

```diff
@@ -10,6 +10,16 @@ on:
         type: string
       get-redis:
         type: boolean
+      test-config:
+        description: 'Test configuration environment variable (e.g. "CONFIG=concurrent_write" or "QUICK=1")'
+        required: true
+        type: string
+      coordinator:
+        type: boolean
+        default: true
+      standalone:
+        type: boolean
+        default: true
```

Comment:
Are these required? when are we using `coordinator` or `standalone` as false?

Reply:
Not required and default to `true`, but if one is false you can test only one of the configurations (the other one).
It is used now for running specific configurations that are only relevant to coordinator.